File: BmiCalc/app/src/main/java/pamo/bmicalc/ui/quiz/QuizQuestionsModel.java
package pamo.bmicalc.ui.quiz;

import android.os.Build;

import androidx.annotation.RequiresApi;

import java.util.ArrayList;
import java.util.List;
import java.util.stream.Collectors;

public class QuizQuestionsModel {
    private List<QuizQuestion> questions = new ArrayList<>();

    public List<QuizQuestion> getQuestions() {
        return questions;
    }

    public void setQuestions(List<QuizQuestion> questions) {
        this.questions = questions;
    }

    public QuizQuestion getQuestionById(int id){
        QuizQuestion qq = null;
        for (QuizQuestion q : questions) {
            if (q.getQuestionId() == id)
                qq = q;
        }
        return qq;
    }
}

File: BmiCalcKotlin/app/src/main/java/pamo/bmicalc_kotlin/ui/quiz/QuizQuestionsModel.kt
package pamo.bmicalc_kotlin.ui.quiz

import java.util.ArrayList

class QuizQuestionsModel {
    var questions: MutableList<QuizQuestion?> = ArrayList()
    fun getQuestionById(id: Int): QuizQuestion? {
        var qq: QuizQuestion? = null
        for (q in questions) {
            if (q!!.questionId == id) qq = q
        }
        return qq
    }
}